src/measure_theory/group/prod.lean

```diff
@@ -236,7 +222,8 @@ begin
 end
 
 /-- This is the computation performed in the proof of [Halmos, §60 Th. A]. -/
-@[to_additive "This is the computation performed in the proof of [Halmos, §60 Th. A]."]
+@[to_additive
+  "This is the computation performed in the proof of [Halmos, §60 Th. A]."]
```

Comment:
I don't quite understand why you changed this. There nothing wrong with it of course, but what's the motivation?

Reply:
No particular reason, it was probably unintentional.